cordova-plugin-outline/apple/src/OutlineTunnel.swift

```diff
@@ -23,9 +23,10 @@ class OutlineTunnel: NSObject, Codable {
   var port: String?
   var method: String?
   var password: String?
+  var prefix: String?
```

Comment:
Should this be a byte array instead?
It's usually better to parse things as soon as possible.

Reply:
OK, done.  I was not able to see a way for this parser to fail, so the parser tolerates arbitrary codepoints and silently ignores the high byte.